### PR TITLE
fix: dagent update self container id & ui updating status

### PR DESCRIPTION
**Pull request by robot9706**
- DAgent now uses `HOSTNAME` env (as a container id then name) to try to find its own container (then tries to parse the CGroup file)
- Added a loading indicator to the node update button when node is updating
- Added an `Update in progress` text to the node connection card as well.

### CODE REVIEW

File: web/crux/proto/crux.proto [hunk 1]
```diff
@@ -824,6 +825,7 @@ message NodeDetailsResponse {
   optional NodeScriptResponse script = 108;
   optional string version = 109;
   NodeType type = 110;
+    bool updating = 111;
```
Comment (nandor-magyar) [inline web/crux/proto/crux.proto:828]: Bad indent

**Review by nandor-magyar — CHANGES_REQUESTED** [review]

File: web/crux/proto/crux.proto (hunk 1 above)
Comment (robot9706) [reply web/crux/proto/crux.proto:828]: oops

**Review by polaroi8d — APPROVED** [review]
Tested local, looks okay for me.

**Review by nandor-magyar — APPROVED** [review]
LGTM

**Pull request merged**